src/client/clientmap.cpp

```diff
@@ -889,10 +883,10 @@ void ClientMap::updateDrawListShadow(const v3f &shadow_light_pos, const v3f &sha
 	// they are not inside the light frustum and it creates glitches.
 	// FIXME: This could be removed if we figure out why they are missing
 	// from the light frustum.
-	for (auto &i : m_drawlist) {
-		i.second->refGrab();
-		m_drawlist_shadow[i.first] = i.second;
-	}
+	// for (auto &i : m_drawlist) {
```

Comment:
Is this no longer needed? (Because culling was removed below?)
If not, let's remove it instead of commenting.

Reply:
No longer needed. I commented it to test and forgot to remove